### PR TITLE
Docket switch queue display updates

### PR DESCRIPTION
**Pull request by leikkisa**
[CASEFLOW-1298](https://vajira.max.gov/browse/CASEFLOW-1298)

### Description
Does not show assigned user to completed docket switch appeal stream, renames status from Archived to Docket Switched and updates copy on the Case List.

### Acceptance Criteria

As a OCB User, I would like the status and assignee to be self-explanatory, so that I can easily understand when a Docket Switch task has been completed.

Given that an OCB user is in their Queue.....

- [x] Verify that the appeal is showing in the completed tab, when the docket switch task has been completed
- [x] Verify that the appeal status showed as "Docket Switch (Inactive) ", when viewing the completed docket switch task.
- [x] Verify that the appeal Assigned To column is empty, when viewing the completed docket switch task.

### Testing Plan
1. Go to ...

### User Facing Changes
 - [ ] Screenshots of UI changes added to PR & Original Issue

 BEFORE|AFTER
 ---|---

### Code Documentation Updates
- [ ] Add or update code comments at the top of the class, module, and/or component.

### Storybook Story
*For Frontend (Presentationa) Components*
* [ ] Add a [Storybook](https://github.com/department-of-veterans-affairs/caseflow/wiki/Documenting-React-Components-with-Storybook) file alongside the component file (e.g. create `MyComponent.stories.js` alongside `MyComponent.jsx`)
* [ ] Give it a title that reflects the component's location within the overall Caseflow hierarchy
* [ ] Write a separate story (within the same file) for each discrete variation of the component

### CODE REVIEW

File: spec/feature/queue/docket_switch_spec.rb [hunk 1]
```diff
@@ -366,6 +366,11 @@
       )
       expect(page).to have_content format(COPY::DOCKET_SWITCH_GRANTED_SUCCESS_MESSAGE)
 
+      # Queue display updates
+      find("span", text: "View all cases").click
+      expect(find_by_id("table-row-1")).to have_content("Docket Switch (Inactive)", appeal.docket_number)
+      expect(page).to have_no_content(COPY::CASE_LIST_TABLE_ASSIGNEE_IS_CURRENT_USER_LABEL)
+
```
Comment (meckila-navapbc) [inline spec/feature/queue/docket_switch_spec.rb:373]: Is there a check for whether or not the task shows in the completed tab?

File: client/app/queue/utils.js [hunk 2]
```diff
@@ -699,9 +699,15 @@ export const labelForLocation = (appeal, userId) => {
...
-  ) : (
-    appeal.status ? StringUtil.snakeCaseToCapitalized(appeal.status) : ''
-  );
+export const statusLabel = (appeal) => {
+  switch(appeal.status) {
+    case 'cancelled':
+      return <span {...css({ color: COLORS.RED })}>{capitalize(appeal.status)}</span>;
+      break;
+    case 'docket_switched':
+      return 'Docket Switch (Inactive)';
```
Comment (meckila-navapbc) [inline client/app/queue/utils.js:708]: We could use copy here if needed so that it can be reused in the spec below

Comment (leikkisa) [reply client/app/queue/utils.js]: Fixed! Had already done that locally but hadn't pushed it :-P

File: spec/feature/queue/docket_switch_spec.rb (hunk 1 above)
Comment (leikkisa) [reply spec/feature/queue/docket_switch_spec.rb:374]: I don't see it so I'll add that, thanks! The functionality for this was already implemented so I didn't think of testing it.

**Review by nanotone — APPROVED** [review]
LGTM, feel free to merge once you're happy with the feature test!

**Review by meckila-navapbc — APPROVED** [review]

**Review by meckila-navapbc — APPROVED** [review]

**Pull request merged**